<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Babel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Enter text:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="inputEdit">
      <property name="plainText">
       <string>Hello World</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Convert from:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sourceComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="targetComboBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="convertButton">
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Result:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="outputEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage_2">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="action"/>
    <addaction name="actionEnglish"/>
    <addaction name="actionSpanish"/>
    <addaction name="actionRussian"/>
    <addaction name="actionArabic"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionabout"/>
    <addaction name="actionTheUserWW"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPhoneticChart"/>
   </widget>
   <addaction name="menuLanguage_2"/>
   <addaction name="menuAbout"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionabout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTheUserWW">
   <property name="text">
    <string>TheUserWW</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>简体中文</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="actionArabic">
   <property name="text">
    <string>العربية</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="text">
    <string>Español</string>
   </property>
  </action>
  <action name="actionPhoneticChart">
   <property name="text">
    <string>Phonetic Chart</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
